config.json

```diff
@@ -4,6 +4,6 @@
     "homepage": "https://jhu.edu",
```

Comment:
Should we change the homepage and logo in config.json?
This file may not be read by the Ember app, though